python/spacewalk/satellite_tools/repo_plugins/__init__.py

```diff
@@ -115,6 +115,14 @@ def set_checksum(self, checksum_type_in=None, checksum_in=None):
             if not((checksum_type_in in self.checksums) and (self.checksums[checksum_type_in] == checksum_in)):
                 self.checksums[checksum_type_in] = checksum_in
 
+    def get_nvra_tuple(self) -> tuple:
+        """returns a tuple of name, version, release, arch"""
+        return (self.name, self.version, self.release, self.arch)
+
+    def is_metadata_valid(self) -> bool:
+        """Returns True if the package NVREA is set"""
+        return all(self.get_nvra_tuple())
```

Comment:
The docstring mentions `NVREA` but `self.get_nvra_tuple` does not include the `E (epoch)` component. Which of the two is it :smile:  ?